### PR TITLE
Readme update for using an existing faucet

### LINKED ISSUE #27: Update readme for using an existing faucet
- Add chain details for which faucet is hosted
- Add tokens supported by faucet

### PR DESCRIPTION
**Pull request by abhayks1**
Fixes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,36 @@ Mosaic Faucet is an ethereum faucet that supports multiple chains simultaneously
...
 ```bash
-curl -H "Content-Type: text/json" -d '{"beneficiary": "0x3e8880f88cf9a146a9d2c5001037d6d963224b8b@3"}' server:port
+curl -H "Content-Type: text/json" -d '{"beneficiary": "{beneficiaryAddress}@5"}' server:port
 ```
 
 Note that the beneficiary is of the format `address@chain`.
 `address` is the address of the beneficiary.
 `chain` is the chain identifier.
-In the example above, chain id `3` refers to ropsten.
+In the example above, chain id `3` refers to goerli.
```
Comment (gulshanvasnani) [inline README.md]:
```suggestion
In the example above, chain id `5` refers to goerli.
```

File: README.md [hunk 2]
```diff
@@ -4,22 +4,36 @@ Mosaic Faucet is an ethereum faucet that supports multiple chains simultaneously
...
 
 The service can run in "coin" or "eip20" mode for a chain.
 It can be different for different chains of the same service.
 In "coin" mode, the faucet will send base coins of the chain to the beneficiary.
 In "eip20" mode, the faucet will send a transaction to the configured EIP20 token to make a transfer to the beneficiary.
 
+### Supported chains and tokens
+
+Faucet is hosted on domain `http://faucet.mosaicdao.org` for below chains.
```
Comment (gulshanvasnani) [inline README.md]:
```suggestion
Faucet is hosted on domain `http://faucet.mosaicdao.org` for goerli and 1405 chains.
```

**Review by gulshanvasnani — COMMENTED** [review]
Nice work. 
Couple of inline comments

File: README.md (hunk 1 above)
Comment (abhayks1) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (abhayks1) [reply README.md]: Done!

**Review by gulshanvasnani — APPROVED** [review]
LGTM

**Pull request merged**